kube/resources/scheduler.yaml

```diff
@@ -193,6 +203,9 @@ spec:
               mountPath: /configs
             - name: airbyte-volume-workspace
               mountPath: /workspace
+            - name: gcs-log-creds-volume
```

Comment:
we should really be using secrets for the db + s3 logs.

Reply:
these are being mounted from a secret - is that what you were thinking of?